src/common/src/types/postgres_type.rs

```diff
@@ -30,21 +30,21 @@ macro_rules! for_all_base_types {
     ($macro:ident $(, $x:tt)*) => {
         $macro! {
             $($x, )*
-            { Boolean     |   16 |     1000 | bool        |      1 }
-            { Bytea       |   17 |     1001 | bytea       |     -1 }
-            { Int64       |   20 |     1016 | int8        |      8 }
-            { Int16       |   21 |     1005 | int2        |      2 }
-            { Int32       |   23 |     1007 | int4        |      4 }
-            { Float32     |  700 |     1021 | float4      |      4 }
-            { Float64     |  701 |     1022 | float8      |      8 }
-            { Varchar     | 1043 |     1015 | varchar     |     -1 }
-            { Date        | 1082 |     1182 | date        |      4 }
-            { Time        | 1083 |     1183 | time        |      8 }
-            { Timestamp   | 1114 |     1115 | timestamp   |      8 }
-            { Timestamptz | 1184 |     1185 | timestamptz |      8 }
-            { Interval    | 1186 |     1187 | interval    |     16 }
-            { Decimal     | 1700 |     1231 | numeric     |     -1 }
-            { Jsonb       | 3802 |     3807 | jsonb       |     -1 }
+            { Boolean     |   16 |     1000 | bool        | boolin         |      1 }
+            { Bytea       |   17 |     1001 | bytea       | byteain        |     -1 }
+            { Int64       |   20 |     1016 | int8        | int8in         |      8 }
+            { Int16       |   21 |     1005 | int2        | int2in         |      2 }
+            { Int32       |   23 |     1007 | int4        | int4in         |      4 }
+            { Float32     |  700 |     1021 | float4      | float4in       |      4 }
+            { Float64     |  701 |     1022 | float8      | float8in       |      8 }
+            { Varchar     | 1043 |     1015 | varchar     | varchar_in     |     -1 }
```

Comment:
`varcharin`

Is this result not copied directly from postgresql output? The macro was written to accept psql output format (`|` separated) as input directly.